cloud/app.py

```diff
@@ -116,7 +138,8 @@ def about():
 
 
 @app.route('/initializebridge', methods=['POST'])
-def initializeBridge():
+@token_required
+def initializeBridge(token_id):
```

Comment:
Why did you introduce the parameter `token_id` in all methods? I can not see you using it anywhere

Reply:
because at the moment the method token_required returns the token id and if we do not introduce it explicitly as a parameter we will get an error

Reply:
But why does the `token_required` method returns it if it is not needed?

Reply:
because the wrapper function is expecting to get something returned and gives an error otherwise

Reply:
Are you sure? It didn't make sense for me so I wrote a minimal example that does not have any errors:

```python
#!/usr/bin/env python3

from functools import wraps

def add_wrapper(func):
    @wraps(func)
    def wrapper():
        print('function was called')
        return func()
    return wrapper

@add_wrapper
def test():
    print('test')
    return 'return'

print(test())
```
The output is (as expected):
```
function was called
test
return
```

Reply:
Did you also change this line: `return f(data['public_id'], *args, **kwargs)`?

Reply:
So it was all copy pasted :joy: